ddtrace/contrib/pymysql/patch.py

```diff
@@ -52,8 +54,15 @@ def _connect(func, instance, args, kwargs):
     return patch_conn(conn)
 
 
+def convert(conn, a):
+    attribute = getattr(conn, a, "")
+    if isinstance(attribute, bytes):
+        attribute = str(attribute, encoding="utf-8")
```

Comment:
We have `ddtrace.internal.compat.ensure_text()` for these kind of things

Reply:
Good to know, thanks I updated the code to use it!